src/builder/command.rs

```diff
@@ -1604,6 +1604,16 @@ impl Command {
     /// strings. After this setting is set, this will be *the only* usage string
     /// displayed to the user!
     ///
+    /// # Multiple usage lines
+    ///
+    /// Multiple usage lines may be present in the usage argument, but some
+    /// rules need to be followed to ensure the usage lines are formatted
+    /// correctly by the default help formatter:
+    ///
+    /// - Do not indent the first usage line.
+    /// - Indent all subsequent usage lines with four spaces.
+    /// - The last line must not end with a newline.
```

Comment:
Generally something like this would be in a `**NOTE:**` rather than getting a `#` heading

Reply:
Fixed.